clients/privacy-center/types/api/models/Identity.ts

```diff
@@ -9,4 +9,5 @@ export type Identity = {
   phone_number?: string;
   email?: string;
   ga_client_id?: string;
+  ljt_readerID?: string;
```

Comment:
Question: Should this be `ljt_reader_id` to be consistent with your existing naming convention?

Reply:
This is an API model generated from the backend's type, but yeah odd choice for the python.